core/unitGet.ts

```diff
@@ -74,7 +74,10 @@ export function getLowestWorthUnit(p: ParticipantInstance, place: Place, include
     return undefined
   } else {
     return units.reduce((a, b) => {
-      if (a.diePriority === b.diePriority && a.takenDamage !== b.takenDamage) {
+      if (a.diePriority === b.diePriority) {
+        if (a.usedSustain !== b.usedSustain) {
+          return a.usedSustain ? b : a
```

Comment:
I'm thinking I skipped over a subtlety here when deciding between a unit that has been repaired this round (used sustain, no damage) vs a unit that came into the round with damage (not used sustain, damage). We'd want to lose the damaged unit and keep the freshly repaired one